ios/Podfile

```diff
@@ -51,6 +51,19 @@ target 'Boilerplate' do
   end
 
   post_install do |installer|
+    installer.pods_project.targets.each do |target|
+      if target.name == 'Flipper'
```

Comment:
Hey @abhishek310001 - Verified locally and this fix works. But it'd be good to add a comment here on what issue we are trying to fix, and possibly add the link from where this fix was obtained (a link to GH/stackoverflow maybe)?

Reply:
Thanks for verifying, will add a comment ASAP.